docs/examples/general/data_loading/external_input.ipynb

```diff
@@ -95,7 +95,7 @@
    "outputs": [],
```

Comment:
<p>The external source <strong>operator</strong> can also accept <strong>GPU data from CuPy </strong>...</p><p>For <strong>the</strong> sake of this example, we will create <strong>an</strong> ExternalInputGpuIterator in such a way that it returns data on the GPU already.</p><p>... decode it outside of DALI on <strong>the</strong> CPU</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA/DALI/pull/1997/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
Done